### PR TITLE
#26 Updating lesson 02 with basic definitions

### PR DESCRIPTION
**Pull request by diegoseso**

### CODE REVIEW

File: LESSON-02/README.md [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+  
+    <?php 
+    $x = 10;  
+    $y = 6;
+    echo $x - $y;
+    ?>
+  
+  Will show 4 on the screen
+  
+  Multiplication is represented by - symbol
```
Comment (marcelavbc) [inline LESSON-02/README.md]: Multiplication is represented by * symbol, no - symbol.

File: LESSON-02/README.md [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+    ?>
+     
+  Will show 5 on the screen 	
+ 
+  Modulus is represented by % symbol
+       
+     <?php 
+     $x = 10;  
+     $y = 2;
+     echo $x / $y;
```
Comment (marcelavbc) [inline LESSON-02/README.md]: echo $x % $y.

File: LESSON-02/README.md [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+  
+      <?php
+      $x = 100;  
+      $y = 99;
+      
+      var_dump($x != $y); 
+      ?>
+  
+  will echo false
+
```
Comment (marcelavbc) [inline LESSON-02/README.md]: Sorry... But should be true, no? 100 is not equal 99...

File: LESSON-02/README.md (hunk 1 above)
Comment (diegoseso) [reply LESSON-02/README.md]: Ok, really nice, you are paying attention!

File: LESSON-02/README.md (hunk 2 above)
Comment (diegoseso) [reply LESSON-02/README.md]: really good, I see you are committed

File: LESSON-02/README.md (hunk 3 above)
Comment (diegoseso) [reply LESSON-02/README.md]: Yes you are right!

**Comment by diegoseso** [comment]
Really good Marcela, this is the way to make a review!

**Comment by diegoseso** [comment]
If you agree to the changes you can approve and I will merge after.

**Pull request merged**